tests/components/heos/test_init.py

```diff
@@ -0,0 +1,89 @@
+"""Tests for the init module."""
+import asyncio
+
+from asynctest import patch
+
+from homeassistant.components.heos import (
+    async_setup, async_setup_entry, async_unload_entry)
+from homeassistant.components.heos.const import DATA_CONTROLLER, DOMAIN
+from homeassistant.components.media_player.const import (
+    DOMAIN as MEDIA_PLAYER_DOMAIN)
+from homeassistant.const import CONF_HOST
+
+
+async def test_async_setup_creates_entry(hass, config):
+    """Test component setup creates entry from config."""
+    assert await async_setup(hass, config)
```

Comment:
Use `async_setup_component` from setup.py instead of `async_setup`.

Reply:
Done.